### ISSUE TITLE
static method 'transitionWithView' on UIView not found

### ISSUE THREAD

**Issue by bsorrentino** [kishikawakatsumi/JavaScriptBridge#4]
Hi

I'm starting to using your 'very useful' framework.

It works great but now i've an unexpected behaviour

I've to call static method 'transitionWithView' on UIView (ie. UIView.transitionWithView) but the result is: 

[TypeError: 'undefined' is not a function (evaluating 'UIView.transitionWithView( view, ...)]

My guess about it is that the method using "blocks" are not exported 

Below the code that i've tested

<h5> Transitions.js </h5>

``` Javascript
var UIViewAnimationOptionTransitionCurlUp          = 3 << 20;
var UIViewAnimationOptionTransitionCurlDown        = 4 << 20;
var UIViewAnimationOptionAllowAnimatedContent      = 1 <<  7;

function curl( view, duration, direction )
{
    var animation = UIViewAnimationOptionTransitionCurlUp;

    if ( direction == "down" )
    {
        animation = UIViewAnimationOptionTransitionCurlDown;
    }

    var animations = function() {};
    var completion = function() {};

    var view = self.viewController.view;


    UIView.transitionWithView( view, duration, (UIViewAnimationOptionAllowAnimatedContent | animation), animations, completion );

}
```

**Comment by kishikawakatsumi** [comment]
You're right.

The methods that includes block parameters must be exported as `JSValue`.
Sorry, but I haven't gotten around to it yet.

I'll investigate and fix it soon...

Thanks for your reporting.

**Comment by kishikawakatsumi** [comment]
Done. https://github.com/kishikawakatsumi/JavaScriptBridge/pull/6

Please check it.

Note:
`transitionWithView〜`method is exported as `transitionWithViewDurationOptionsAnimationsCompletion` followed by standard JavaScriptCore's conventions. 

Example:

``` javascript
var UIViewAnimationOptionTransitionCurlUp          = 3 << 20;
var UIViewAnimationOptionTransitionCurlDown        = 4 << 20;
var UIViewAnimationOptionAllowAnimatedContent      = 1 <<  7;

var animation = UIViewAnimationOptionTransitionCurlUp;

var animations = function() {};
var completion = function() {};

var view = self.view;

UIView.transitionWithViewDurationOptionsAnimationsCompletion(view, 0.5, (UIViewAnimationOptionAllowAnimatedContent | animation), animations, completion);
```

**Comment by bsorrentino** [comment]
Hi

It works perfectly now

thanks for quick & elegant fix

Note: 
Hope that you update & release podspec soon

**Comment by kishikawakatsumi** [comment]
Released 0.1.0 https://github.com/CocoaPods/Specs/pull/7283
